third_party/googleapis/README.google

```diff
@@ -8,11 +8,15 @@ Local Modifications:
        README.google was removed from .gitignore.
        Remote submodules were not imported.
        Remove .gitmodules.
+       Remove submodules from git index with 'git rm'.
 
        Imported as a git-subtree.
 
        $ git remote add googleapis https://github.com/googleapis/googleapis
        $ git subtree add --prefix third_party/googleapis googleapis master --squash
+       $ rm third_party/googleapis/.gitmodules
+       $ # For all submodules:
+       $ git rm third_party/googleapis/{path_to_submodule}
```

Comment:
Could this use "git submodule foreach"?

Reply:
I am just not sure what "For all submodules" means here. Is this command something that needs to be run every time the subtree needs to be updated? If so, the instructions on what "all" means here should be clearer (ideally just some command that gathers that). If this text is about the one time subtree addition that was done, then perhaps list explicit submodules you removed?

Reply:
It's a one time setup. It's sufficient to follow "To update from remote on an ongoing basis" instructions below to update the subtree.

Sure, updated the PR to list explicit submodules removed.